vendor/cookbooks/ruby/attributes/binary.rb

```diff
@@ -0,0 +1,5 @@
+case node["platform_family"]
+when 'debian'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.